### PR TITLE
Wrapper for init-metadata

### PR DESCRIPTION
**Pull request by siva-nb**
Fixes https://github.com/apache/druid/issues/13738

Description
As described part of the issue, created a wrapper shell script with druid database init scripts

Release note
Wrapper for druid database init scripts

Replace [13756](https://github.com/apache/druid/pull/13756)
This PR has:

- [X] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [X] added documentation for new or modified features or behaviors.
- [ ] a release note entry in the PR description.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [X] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

**Comment by siva-nb** [comment]
@LakshSingla I have closed the previous PR - https://github.com/apache/druid/pull/13756 and addressed the review comments in this PR. Kindly take a look.

File: examples/bin/init-metadata.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Tool for creating Druid's metadata tables
+# Usage: init-metadata [-d|c|u|p|v]
+
+
+#params : type loadList 
+setup()
+{
+  cd ${DRUID_ROOT}
+  java -classpath "lib/*" -Dlog4j.configurationFile=conf/druid/cluster/_common/log4j2.xml -Ddruid.extensions.directory="extensions" -Ddruid.extensions.loadList="[\"$2\"]" -Ddruid.metadata.storage.type=$1 -Ddruid.node.type=metadata-init org.apache.druid.cli.Main tools metadata-init --connectURI=$connectURI --user $username --password $password --base druid
```
Comment (LakshSingla) [inline examples/bin/init-metadata.sh:28]: * Please convert `java` into a configurable variable.
* Let's not `cd` into the directory, rather prepend all the paths with this.

**Comment by LakshSingla** [comment]
Can you please share the steps that you used to test the script's function? The couple of failing tests are unrelated.

**Comment by siva-nb** [comment]
@cryptoe @LakshSingla thanks for your comments, will address those in a day or two. Thanks

**Pull request closed without merging**